frontend/src/store/modules/v1/projectIamPolicy.ts

```diff
@@ -31,7 +31,7 @@ export const useProjectIamPolicyStore = defineStore(
         list.push(...binding.members);
         return list;
       }, [] as string[]);
-      await batchGetOrFetchUsers(members);
+      await useUserStore().batchGetUsers(members);
```

Comment:
The members here will be in `user:email` format

Reply:
Updated